README.md

```diff
@@ -391,13 +395,43 @@ impl PyIterProtocol for MyCoroutine {
     }
 }
 
+// Exposing our custom awaitable to Python.
+// This will behave like a coroutine.
+#[pyfunction]
+fn my_coroutine() -> PyResult<MyCoroutine> {
+    Ok(MyCoroutine {})
+}
```

Comment:
Either that or we could expose a constructor for `MyCoroutine`. I thought this was more in line with the spirit of the previous examples, but it's your call.

Reply:
I think it would be a good idea to remove the `PyResult<>` in this case it isn't needed so probably best to keep it simple.

Reply:
Didn't know you could do that. Even cleaner that way! Commit amended